src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -55,6 +65,11 @@ export class CalciteColorPicker {
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * When true, the color picker will process and display alpha characters.
+   */
+  @Prop() alphaSupport = false;
```

Comment:
> I'm just hoping we can not start a practice of using words like enabled/support

I get where you're coming from and I agree that a prefix/suffix convention isn't always desirable, but I think it will be fine as long as the boolean name expresses its intent. Single-word boolean props can read like they modify something unrelated or accept a value:

```html
<!-- the picker is transparent? -->
<calcite-color-picker transparent ></ calcite-color-picker>

<!-- can I set the transparency value? -->
<calcite-color-picker transparency ></ calcite-color-picker>
```

I'll update https://github.com/Esri/calcite-components/pull/2945 accordingly since it originally aimed at establishing a naming pattern that won't always be applicable.